app/src/test/contrib/stores/ResponsibilityStoreTests.ts

```diff
@@ -75,19 +74,24 @@ describe("ResponsibilityStore", () => {
         expect(responsibilityStore.getState().currentModellingGroup).to.eql(group);
     });
 
-    it("responsibilityActions.update sets responsibility set", () => {
+    it("responsibilityActions.update sets current responsibility set", () => {
```

Comment:
Actually we were both confused here, but I think the code is right as it stands. We just confused the action name with the sentence.